client/blocks/product-selector/index.jsx

```diff
@@ -84,12 +82,26 @@ export class ProductSelector extends Component {
 		} );
 	}
 
+	getProductOptions( product ) {
+		const { intervalType, storeProducts } = this.props;
+		const productSlugs = product.options[ intervalType ];
+
+		return productSlugs.map( productSlug => {
+			const productObject = storeProducts[ productSlug ];
```

Comment:
Do we need to tighten it here? Is there any chance that this won't exist?

Reply:
That's a good question. I think it's a good idea to make sure it exists. It may not be there if a non-existent product is passed to the `<ProductSelector />` component. I'll fix this code.

Reply:
As discussed on Slack, we'll address this issue in a separate PR.